smrf/tests/mock_helpers.py

```diff
@@ -0,0 +1,57 @@
+from collections import OrderedDict
+
+import numpy as np
+
+import smrf
+
+
+class MockTopo(smrf.data.load_topo.Topo):
```

Comment:
This is the upper 4x4 grid from the RME topo. I liked the idea of having the topo mocked out for the smaller tests.